README.md

```diff
@@ -105,6 +105,14 @@ let viewController = appResolver.resolveViewController(user: user)
 
 Install code generator `dikitgen` first.
 
+### [Mint](https://github.com/yonaskolb/mint)
+
+```shell
+mint run ishkawa/DIKit dikitgen
```

Comment:
Looks like this should be `mint install` for installation.

Reply:
Thank you for your review. I fixed from `run` to `install` by your comment.